### PR TITLE
Use Liquibase for DB migrations

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-949

### CODE REVIEW

File: FFCDemoPaymentService/Data/ApplicationDbContext.cs [hunk 1]
```diff
@@ -6,11 +6,20 @@ namespace FFCDemoPaymentService.Data
...
+        public ApplicationDbContext() { }
         public ApplicationDbContext(PostgresConnectionStringBuilder stringBuilder) : base()
         {
             connectionStringbuilder = stringBuilder;
+            schemaConfig = null;
+        }
+
+        public ApplicationDbContext(PostgresConnectionStringBuilder stringBuilder, SchemaConfig schemaConfig) : base()
+        {
+            connectionStringbuilder = stringBuilder;
```
Comment (markysoft) [inline FFCDemoPaymentService/Data/ApplicationDbContext.cs:21]: bit of a nitpick, as was kind of an existing problem, but the 'b' in 'string builder' isn't consistently capitalised, sometimes it's `connectionStringbuilder` and other times it's `stringBuilder`

**Review by markysoft — COMMENTED** [review]
looks good, verified tables are being created, just a small nitpick

File: FFCDemoPaymentService/Data/ApplicationDbContext.cs (hunk 1 above)
Comment (st3v3nhunt) [reply FFCDemoPaymentService/Data/ApplicationDbContext.cs]: :+1: I've made the change.

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**